lib/fluent/plugin/out_kafka2.rb

```diff
@@ -286,6 +292,27 @@ def write(chunk)
           topic = @default_topic
           retry
         end
+        if @create_topic
+          producer.shutdown if producer
+          topic_template = chunk.metadata.variables[@topic_template_key_sym]
```

Comment:
`@topic_template_key.to_sym` ?

Reply:
Oh, it's defined at https://github.com/fluent/fluent-plugin-kafka/pull/407/files#diff-d2728a692f5968218f3d1a0ab0eb4145cb76491cbc1a2a6ce697c84a4efcd1c7R175.